src/components/Select/Select.module.scss

```diff
@@ -0,0 +1,102 @@
+/* Same as Type.scss Body. Should we use a mixin? */
```

Comment:
Could probably also use CSS variables

Reply:
I'll look into that tomorrow 👍 

Also, I need to work on this a bit more:

![image](https://user-images.githubusercontent.com/142403/69214915-1dcaab80-0b1d-11ea-9050-caf3cae29f79.png)

I removed margin-top but that won't work for above use case where we wrap (the screengrab has the 2px margin-top so I'll have to add that back and then work out another way to get things looking correct)

Reply:
Ok, 65e937bc55b5ed9e23c17794fe0c144b7a4434de should address all of these